cli/running/running.go

```diff
@@ -225,6 +323,18 @@ func createPIDFile(pidFileName string) error {
 	//    user:   read/write
 	//    group:  read
 	//    others: read
+	pidAbsDir := filepath.Dir(pidFileName)
```

Comment:
Looks like, the comment above must be moved to 337 line. Because it is about the file creation, not a directory.

Reply:
fixed